meetup/services/meetup_presenter.py

```diff
@@ -0,0 +1,42 @@
+import time
+from pizzaplace.services.pizza_place_presenter import PizzaPlacePresenter
+from pizzaplace.services.yelp_api import YelpApi
+
+class MeetupPresenter():
+  def __init__(self, meetup, meetup_api, api_response_parser):
+    self.meetup = meetup
+    self.parsed_api_response = self.get_api_response(meetup_api, api_response_parser)
+
+  def get_meetup_link(self):
```

Comment:
It's OK to abandon the get/set convention from Java and be a little more terse. The purpose of this presenter is to create a friendly interface for presenting data about the meetup that will read nicely when used in a template. Does including `get_` in the method name help or hinder that goal?